bin/codemods/README.md

```diff
@@ -4,6 +4,30 @@
 
 Code modification scripts, also known as codemods, are transformation scripts that can simultaneously modify multiple files with precision and reliability. Codemods were popularized by [Facebook's engineering team](https://medium.com/@cpojer/effective-javascript-codemods-5a6686bb46fb) and depends greatly on Facebook's [jscodeshift](https://github.com/facebook/jscodeshift) library, which wraps over a library named [recast](https://github.com/benjamn/recast) (author of which is associated with the [Meteor](https://www.meteor.com/) project).
 
+## How to write codemods
+
+Place your codemod under `src` folder:
+```bash
+touch ./bin/codemods/src/your-transformation-name.js
+```
+
+Here's a stub to begin with:
+```js
+const config = require( './config' );
+
+export default function transformer( file, api ) {
+	const j = api.jscodeshift;
+	const root = j( file.source );
+
+	// Modify file's AST (Abstract Syntax Tree) structure here
+
+	return root.toSource( config.recastOptions );
+}
+```
+
+A nifty tool to explore AST structures is [AST explorer](https://astexplorer.net/).
+You can choose "esprima" as a parser as that's what Recast uses internally.
```

Comment:
Hmm, it's set to `recast` for me by default. Isn't that the better option?
Also, shouldn't we advise to pick `jscodeshift` from the 'Transform' menu? :slightly_smiling_face:

Reply:
Oh! Funny, `acorn` by default for me. :-) Recast would indeed be better!

Reply:
@ockham Updated! :thumbsup: